addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/NetatmoBindingConstants.java

```diff
@@ -168,8 +164,7 @@
             .concat(SUPPORTED_DEVICE_THING_TYPES_UIDS.stream(), Stream.of(APIBRIDGE_THING_TYPE))
             .collect(Collectors.toSet());
 
-    public static final Set<String> MEASURABLE_CHANNELS = Stream.of(CHANNEL_BOILER_ON, CHANNEL_BOILER_OFF)
-            .collect(Collectors.toSet());
+    public static final Set<String> MEASURABLE_CHANNELS = Collections.emptySet();
```

Comment:
Is there a reason to keep this static final? If so this won't work: `Collections.emptySet()` returns an immutable set.

Reply:
I kept it in case we need it one day but ok I could suppress it.

Reply:
@clinique : this set is now empty. Is the class MeasurableChannels still necessary ?